tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -368,6 +391,34 @@ private void RunWaf(object sender, InstrumentationGatewaySecurityEventArgs e)
             }
         }
 
+        public void ProcessControlCommand(object arg)
```

Comment:
As discussed, good strategy for testing, but must be removed before merging.

Reply:
Done